curl, wget: Complete IP addresses, including ipv6 for --interface and --bind-address

yedayak opened the pull request:
Closes https://github.com/scop/bash-completion/issues/1315

SoniEx2 commented:
looks good! tho there is an issue with the wget completion where we can start the completion but not finish it. the curl works perfectly tho! thanks ^^

SoniEx2 commented:
(ah we see, it seems that is answered in the README. unsure why it works for curl tho.)

yedayak commented:
> (ah we see, it seems that is answered in the README. unsure why it works for curl tho.)

I'm not sure either, bu a workaround is putting the 

> (ah we see, it seems that is answered in the README. unsure why it works for curl tho.)

Yeah I missed this about `_comp_compgen_ip_addresses`:
```
# If producing IPv6 completions, `_comp_initialize` with `-n :`.
```
Fixed, and added tests

yedayak commented:
@SoniEx2 Does this fix the issue?

SoniEx2 commented:
yes! thanks!
